### PR TITLE
[New Concept] General: CMS

### PR DESCRIPTION
**Pull request by dharmelolar**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [ ] All writings are my own.
- [ ] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/cms/cms.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+​A content management system (CMS) is a software application that allows users to create, edit, and manage digital content such as text, images, videos, and other multimedia files. A CMS typically provides a user-friendly interface that allows users to easily add, delete, and modify content without the need for technical knowledge or coding skills. The primary purpose of a CMS is to make it easier for organizations to manage and maintain their online presence by providing a central platform for organizing and publishing content.
```
Comment (SSwiniarski) [inline content/general/concepts/cms/cms.md:1]: We're missing a metadata section

Comment (SSwiniarski) [inline content/general/concepts/cms/cms.md:1]:
```suggestion
​A **content management system (CMS)** is a software application that allows users to create, edit, and manage digital content such as text, images, videos, and other multimedia files. A CMS typically provides a user-friendly interface that allows users to easily add, delete, and modify content without the need for technical knowledge or coding skills. The primary purpose of a CMS is to make it easier for organizations to manage and maintain their online presence by providing a central platform for organizing and publishing content.
```

File: content/general/concepts/cms/cms.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
```
Comment (SSwiniarski) [inline content/general/concepts/cms/cms.md:2]: We should only have a single line break between sections

File: content/general/concepts/cms/cms.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+A CMS typically has two main components: a content management application (CMA) and a content delivery application (CDA). The CMA is the user interface that allows users to create and manage content, while the CDA is the backend that handles the delivery of the content to the website or other digital platforms. Some common features of a CMS include managing and organizing digital assets, creating and editing content, and controlling access to different users.
+
+## Common examples of CMS
+
+- **WordPress**: is a free and open-source CMS based on PHP and MySQL and It is the most popular CMS in the world.WordPress offers a wide range of features, including customizable themes, plugins, and widgets, as well as the ability to manage multiple users and content types. 
+
```
Comment (SSwiniarski) [inline content/general/concepts/cms/cms.md:14]:
```suggestion
```

File: content/general/concepts/cms/cms.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+A CMS typically has two main components: a content management application (CMA) and a content delivery application (CDA). The CMA is the user interface that allows users to create and manage content, while the CDA is the backend that handles the delivery of the content to the website or other digital platforms. Some common features of a CMS include managing and organizing digital assets, creating and editing content, and controlling access to different users.
+
+## Common examples of CMS
+
+- **WordPress**: is a free and open-source CMS based on PHP and MySQL and It is the most popular CMS in the world.WordPress offers a wide range of features, including customizable themes, plugins, and widgets, as well as the ability to manage multiple users and content types.
```
Comment (SSwiniarski) [inline content/general/concepts/cms/cms.md:13]:
```suggestion
- **WordPress**: is a free and open-source CMS based on PHP and MySQL and It is the most popular CMS in the world. WordPress offers a wide range of features, including customizable themes, plugins, and widgets, as well as the ability to manage multiple users and content types. 
```

File: content/general/concepts/cms/cms.md (hunk 1 above)
Comment (dharmelolar) [reply content/general/concepts/cms/cms.md:1]: @SSwiniarski how do I get the metadata?

Comment (SSwiniarski) [reply content/general/concepts/cms/cms.md:1]: Look at the metadata section in the content standards [here](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) to see what goes in the metadata, then look at the source of the entry template [here](https://github.com/Codecademy/docs/blob/main/documentation/entry-template.md) to see how to format it in the file.

Comment (SSwiniarski) [reply content/general/concepts/cms/cms.md:1]: You can also look at the source of a similar General entry to get an idea what to add.

Comment (dharmelolar) [reply content/general/concepts/cms/cms.md]: Done!

File: content/general/concepts/cms/cms.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  
+CatalogContent:
+ - 'learn-html'
+ - 'learn-css'
+ - 'paths/front-end-engineer-career-path'
+---
+
+​A **Content Management System (CMS)** is a software application that allows users to create, edit, and manage digital content such as text, images, videos, and other multimedia files. A CMS typically provides a user-friendly interface that allows users to easily add, delete, and modify content without the need for technical knowledge or coding skills. The primary purpose of a CMS is to make it easier for organizations to manage and maintain their online presence by providing a central platform for organizing and publishing content.
+
+## How does a content Management System Work?
```
Comment (KTom101) [inline content/general/concepts/cms/cms.md:19]: subheadings should be titled cased for docs entries.  Also, try staying away from question-based subheadings in docs.
```suggestion
## CMS Features
```

File: content/general/concepts/cms/cms.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+A content management system provides a user-friendly interface that allows non-technical users to easily create, edit, and publish content to a website without having to write any code. The CMS manages all of the underlying technical aspects of the website, such as creating and storing the content in a database, generating the necessary HTML code to display the content on the website, and handling user permissions and other security issues.
+
+A CMS typically has two main components: a content management application (CMA) and a content delivery application (CDA). The CMA is the user interface that allows users to create and manage content, while the CDA is the backend that handles the delivery of the content to the website or other digital platforms. Some common features of a CMS include managing and organizing digital assets, creating and editing content, and controlling access to different users.
+
+## Common examples of CMS
```
Comment (KTom101) [inline content/general/concepts/cms/cms.md:25]:
```suggestion
## Common Examples of CMS
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@dharmelolar I've added two changes pertaining to the subheadings (please feel free to change the wording around). However, I also noticed you still have one change that needs to be done from the previous review which is the removal of the file `tech-stacks.md`, did you need any help with this?

**Comment by SSwiniarski** [comment]
@dharmelolar just a reminder that you still need to remove the tech-stacks.md file from this branch.

**Comment by dharmelolar** [comment]
@SSwiniarski I've made all the necessary corrections!

**Review by KTom101 — APPROVED** [review]
@dharmelolar LGTM 👍

@SSwiniarski Anything left to add?

**Review by SSwiniarski — APPROVED** [review]
@dharmelolar, Looks good from here.

**Comment by KTom101** [comment]
@dharmelolar Thank you for contributing. The entry is live now: https://www.codecademy.com/resources/docs/general/cms
Screen Shot 2023-01-08 at 1 11 59 PM

**Pull request merged**